### PR TITLE
Update exporter README.md

### PR DESCRIPTION
**Pull request by flands**
**Documentation:** Refresh documentation so it is consistent and add missing exporter documentation

### CODE REVIEW

File: exporter/README.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
-# Exporters
+# General Information
 
-Below is the list of exporters directly supported by the OpenTelemetry Collector.
+An exporter is how data gets sent to different systems/back-ends. Generally, an
+exporter translate the internal format into another defined format.
```
Comment (tigrannajaryan) [inline exporter/README.md]:
```suggestion
exporter translates the internal format into another defined format and sends the data elsewhere (typically via network).
```

File: exporter/README.md [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+- [Zipkin](#zipkin)
+
+Supported metric exporters (sorted alphabetically):
+
+- [OpenCensus](#opencensus)
+- [Prometheus](#prometheus)
+
+Supported local exporters (sorted alphabetically):
+
+- [Logging](#logging)
```
Comment (tigrannajaryan) [inline exporter/README.md]: Also add File Exporter.

Comment (flands) [reply exporter/README.md]: Done

File: exporter/README.md [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
+
+- [OpenCensus](#opencensus)
+- [Prometheus](#prometheus)
+
+Supported local exporters (sorted alphabetically):
+
+- [File](#file)
+- [Logging](#logging)
 
 The [contributors repository](https://github.com/open-telemetry/opentelemetry-service-contrib)
```
Comment (pjanotti) [inline exporter/README.md]:
```suggestion
The [contributors repository](https://github.com/open-telemetry/opentelemetry-collector-contrib)
```

File: exporter/README.md [hunk 4]
```diff
@@ -1,12 +1,23 @@
...
  has more exporters that can be added to custom builds of the service.
```
Comment (pjanotti) [inline exporter/README.md]:
```suggestion
 has more exporters that can be added to custom builds of the Collector.
```

**Pull request merged**